languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10084,6 +10084,378 @@ though, and has since been slightly modified)-->
+                      <token>-</token>
+                      <token>à</token>
+                      <token>-</token>
+                      <token>dire</token>
+                  </marker>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1, \2\3\4\5\6\7</suggestion>
+              <example correction="temps, c'est-à-dire">Il ne réagit pas au <marker>temps c'est-à-dire</marker> aux saisons qui passent.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token>être</token>
+                  <token>soit</token>
+                  <token max="1" postag="M fin" spacebefore="no">.</token>
+              </antipattern>
+              <pattern>
+                  <marker>
+                      <token postag="V.*|R dem m s|A.*|J.*" postag_regexp="yes"><exception regexp="yes">ne|ce|ça|cela|alors|bien</exception></token>
+                      <token>soit</token>
+                  </marker>
+                  <token postag="M fin" spacebefore="no">.<exception scope="next">.</exception></token>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1, \2</suggestion>
+              <example correction="arrive, soit">Il <marker>arrive soit</marker>.</example>
+              <example correction="plus, soit">Il ne le voit <marker>plus soit</marker>.</example>
+              <example>Il ne le voit <marker>plus soit</marker>...</example>
+              <example>Il le faut, <marker>alors soit</marker>.</example>
+              <example>Il n'a de conflit avec qui que <marker>cela soit</marker>.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token postag="SENT_START"/>
+                  <token>soit</token>
+                  <token regexp="yes" skip="-1">[^,;:]+</token>
+                  <token>soit</token>
+              </antipattern>
+              <pattern>
+                  <token postag="SENT_START"/>
+                  <marker>
+                      <token>soit</token>
+                      <token regexp="yes">[^,);:(«»-]+<exception regexp="yes">[a-z]$|[A-Z]$</exception></token></marker>
+              </pattern>
+              <message>Une virgule est conseillée dans ce contexte.</message>
+              <suggestion>\2, \3</suggestion>
+              <example correction="Soit, il"><marker>Soit il</marker> peut venir avec nous.</example>
+              <example correction="Soit, prendre"><marker>Soit prendre</marker> la vie côté plaisir est un concept intéressant.</example>
+              <example correction="Soit, grand"><marker>Soit grand</marker> ou petit, peu m'importe.</example>
+              <example><marker>Soit V</marker> le centre du cercle.</example>
+              <example><marker>Soit grand</marker> soit petit, peu m'importe.</example>
+          </rule>
+          <rule>
```

Comment:
Nice approach to the problem! The results are looking good, but you'll see some residual FPs:

> Pour que "A ou B" soit vrai, il suffit que A soit vrai ou que B soit vrai.